tests/models/test_main.py

```diff
@@ -215,6 +219,7 @@ def test_framework_serialization_with_default_datetimes(self):
             slug='foo-109',
             framework='g-cloud',
             status='open',
+            has_further_competition=True,
```

Comment:
In the previous test we provide both args (with `True` and `False`) but here it's just one - not fussed which way but we should probably be consistent.

Reply:
I purposely left the latter out actually, to also test the default value as a little extra. Maybe I could do this more explicitly?

Reply:
Up to you really!